Source/Editor/Windows/SceneTreeWindow.cs

```diff
@@ -346,6 +346,29 @@ public override bool OnMouseUp(Float2 location, MouseButton buttons)
 
             return false;
         }
+        
+        /// <inheritdoc />
+        public override bool OnMouseDoubleClick(Float2 location, MouseButton button)
+        {
```

Comment:
This is based on the double-click within a certain timeframe and just uses the built-in call for that.

Reply:
Would it make more sense to use `OnMouseDoubleClickHeader` override in `ActorTreeNode`? I guess it could be called in Prefab window too. Just an optional suggestion/idea.

Reply:
Will look into it tonight as I think that is a good suggestion